### PR TITLE
[2.13.x] DDF-04487 Solr stop key for *nix

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
The Solr start script generates a stop key every time Solr is started

#### Who is reviewing it? 
@garrettfreibott  @mcalcote @brjeter @tyler30clemens @austinsteffes 

#### Select relevant component teams: 
@codice/security 
@codice/solr 
#### Ask 2 committers to review/merge the PR and tag them here.
@clockard 
@brjeter 
@garrettfreibott 

For GH Issues:
Fixes: #4487 

####Testing
* Should be tested on both **Mac and Linux**
* Test `ddfsolr start`,  `ddfsolr restart`,  and `ddfsolr stop`
* Test that `ddfsolr stop` removes the file named STOPKEY
* Test that the Solr process is eventually killed even if the STOPKEY file is missing (usually 180 seconds) 
                  
#### Checklist:
- [No] Documentation Updated
- [No] Update / Add Threat Dragon models
- [No] Update / Add Unit Tests
- [No] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an inline code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by austinsteffes** [comment]
hero'd 👌 👌 ✅

File: distribution/ddf-common/src/main/resources/bin/ddfsolr [hunk 1]
```diff
@@ -72,16 +92,15 @@ print_messages() {
...
-    $SOLR_EXEC stop -p $SOLR_PORT
+if [ "$COMMAND" == "stop" ]; then
+	stop_solr
 else
     # Read protocol from properties file
     SOLR_PROTOCOL=$($GET "solr.http.protocol")
 
     # Do not proceed if required properties are missing
     if [ -z "$SOLR_PROTOCOL" ]; then
-        echo "Cannot determine Solr protocol (http or https) from custom.system.properties file. Exiting."
```
Comment (mcalcote) [inline distribution/ddf-common/src/main/resources/bin/ddfsolr]: Looks like `Exiting` accidentally got changed.

Comment (ahoffer) [reply distribution/ddf-common/src/main/resources/bin/ddfsolr]: It's the new way to terminate a batch file. You don't exit it, you exii it.

**Comment by ahoffer** [comment]
I manually tested on Linux after pushing the last two commits.

**Comment by austinsteffes** [comment]
hero'd on mac. checks out

**Comment by ahoffer** [comment]
build now

**Pull request merged**